.github/workflows/fabric_acctest.yml

```diff
@@ -193,3 +203,13 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           files: ./coverage_pfcr.txt
+
+      - name: Create Go Coverage HTML File
```

Comment:
It looks like the coverage is already being uploaded to CodeCov; what's the use case for also creating an actions artifact?

Reply:
When I was first looking at the page that it was being uploaded to I was getting some weird errors and wasn't able to navigate the page. Looks like it's cleared up so I agree the redundancy isn't necessary.

Is there a way to output just the `coverage.txt` file from codecov if we ever wanted to use the `go tool` for coverage reports in html?